packages/boot/test/bootstrapTests/test-liquidation-concurrent-1.ts

```diff
@@ -0,0 +1,557 @@
+
+    const minter =
+      await walletFactoryDriver.provideSmartWallet('agoric1minter');
+
+    for (let i = 0; i < setups[collateralBrandKey].vaults.length; i += 1) {
+      const offerId = `open-${collateralBrandKey}-vault${i}`;
+      await minter.executeOfferMaker(Offers.vaults.OpenVault, {
+        offerId,
+        collateralBrandKey,
+        wantMinted: setups[collateralBrandKey].vaults[i].ist,
+        giveCollateral: setups[collateralBrandKey].vaults[i].atom,
+      });
+      t.like(minter.getLatestUpdateRecord(), {
+        updated: 'offerStatus',
+        status: { id: offerId, numWantsSatisfied: 1 },
+      });
+    }
+
+    // Verify starting balances
+    for (let i = 0; i < setups[collateralBrandKey].vaults.length; i += 1) {
+      check.vaultNotification(managerIndex, i, {
+        debtSnapshot: {
+          debt: { value: scale6(setups[collateralBrandKey].vaults[i].debt) },
+        },
+        locked: { value: scale6(setups[collateralBrandKey].vaults[i].atom) },
+        vaultState: 'active',
+      });
+    }
+  };
+
+  const placeBid = async (collateralBrandKey: string) => {
+    await buyer.sendOffer(
+      Offers.psm.swap(
+        agoricNamesRemotes,
+        agoricNamesRemotes.instance['psm-IST-USDC_axl'],
+        {
+          offerId: `print-${collateralBrandKey}-ist`,
+          wantMinted: 1_000,
+          pair: ['IST', 'USDC_axl'],
+        },
+      ),
+    );
+
+    const maxBuy = `10000${collateralBrandKey}`;
+
+    for (let i = 0; i < setups[collateralBrandKey].bids.length; i += 1) {
+      const offerId = `${collateralBrandKey}-bid${i + 1}`;
+      // bids are long-lasting offers so we can't wait here for completion
+      await buyer.sendOfferMaker(Offers.auction.Bid, {
+        offerId,
+        ...setups[collateralBrandKey].bids[i],
+        maxBuy,
+      });
+      t.like(readLatest('published.wallet.agoric1buyer'), {
+        status: {
+          id: offerId,
+          result: 'Your bid has been accepted',
+          payouts: undefined,
+        },
+      });
+    }
+  };
+
+  const {
+    collateralBrandKey: collateralBrandKeyA,
+    managerIndex: managerIndexA,
+  } = cases[0];
+  const {
+    collateralBrandKey: collateralBrandKeySt,
+    managerIndex: managerIndexSt,
+  } = cases[1];
+
+  await setupVault(collateralBrandKeyA, managerIndexA);
+  await setupVault(collateralBrandKeySt, managerIndexSt);
+
+  await Promise.all(
+    cases.map(({ collateralBrandKey }) => {
+      placeBid(collateralBrandKey);
+    }),
+  );
+
+  // ---------------
+  //  Change price to trigger liquidation
+  // ---------------
+  console.log('Change prices');
+  await priceFeedDrivers[collateralBrandKeyA].setPrice(
+    setups[collateralBrandKeyA].price.trigger,
+  );
+  await priceFeedDrivers[collateralBrandKeySt].setPrice(
+    setups[collateralBrandKeySt].price.trigger,
+  );
+
+  const liveSchedule = readLatest('published.auction.schedule');
+
+  cases.map(async ({ collateralBrandKey, managerIndex }) => {
```

Comment:
`map` is for producing new values. If you're not consuming the value, iterate with a `for` loop.

Though in this case I think you meant to gather a list of promises and away them all?

I'm going to block on this change as it may affect the outcome of the test